src/components/App.jsx

```diff
@@ -1,4 +1,4 @@
-import { instanceOf } from 'prop-types';
+import { PropTypes, instanceOf } from 'prop-types';
```

Comment:
This isn't actually what I meant, maybe it's ok...
I meant you define a properties alias/interface like [here](https://github.com/aertslab/SCope/pull/363/files#diff-cc751c9a9e01ff5fb6d73e2e6128074aR28) and parameterise `Component` with that type.

I'm not actually sure if `PropTypes` does validation. Do you know?

Reply:
Oh I'm an idiot. Sorry I didn't realise this was still JS, of course! In that case this is fine.